### PR TITLE
Adds Table components + unit tests

### PR DESCRIPTION
**Pull request by cfrancisco**
In addition, it adds a simple config to run all unit tests using: "yarn test-all"

### CODE REVIEW

File: src/Components/CustomMap/CustomMap.js [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
-    width: '100%',
-    height: '300px',
-};
+import Util from '../../utils';
 
 
 const CustomMap = ({ markersData, center, zoom }) => {
     // create map
-    const mapRef = useRef(null);
+    const sid = Util.sid();
```
Comment (nunesjoab) [inline src/Components/CustomMap/CustomMap.js]: Could be mapId?

File: src/Pages/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -27,84 +40,184 @@ const styles = () => ({
...
+            layout.push(
+                this.generateDOM(el,
+                    <span className="text">{el.i}</span>),
+            );
+        });
+        this.setState({ layoutElement: layout });
+    }
+
 
+    onRemoveItem(event, index) {
```
Comment (nunesjoab) [inline src/Pages/Dashboard/Dashboard.js:116]: We can start to write a small documentation to our functions. A short description that contains the objective, parameters and return of function. What do you think?

File: src/Pages/Dashboard/Dashboard.js [hunk 3]
```diff
@@ -27,84 +40,184 @@ const styles = () => ({
...
+                y: Math.floor(i / 6),
+                w: 1 + Math.floor(Math.random() * 2),
+                h: 1 + Math.floor(Math.random() * 2),
+            })),
+            values: { element: '' },
+            newCounter: 0,
+            layoutElement: [],
+        };
+
+        this.generateDOM = this.generateDOM.bind(this);
```
Comment (nunesjoab) [inline src/Pages/Dashboard/Dashboard.js:70]: I will replace my binds by arrow function syntax. What do you think?

File: src/Pages/Tables/Tables.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import SimpleTable from '../../Components/Table/SimpleTable';
+
+
+const styles = (theme) => ({
```
Comment (nunesjoab) [inline src/Pages/Tables/Tables.js:7]: Should be placed under Tables folder named as _styles.js?

**Review by nunesjoab — CHANGES_REQUESTED** [review]
Small changes and some questions to clarify some things. Nothing urgent

File: src/Components/CustomMap/CustomMap.js (hunk 1 above)
Comment (cfrancisco) [reply src/Components/CustomMap/CustomMap.js]: It's already mapid. The file is outdated.

File: src/Pages/Dashboard/Dashboard.js (hunk 3 above)
Comment (cfrancisco) [reply src/Pages/Dashboard/Dashboard.js:70]: Is possible, I think is a dev preference.

File: src/Pages/Tables/Tables.js (hunk 4 above)
Comment (cfrancisco) [reply src/Pages/Tables/Tables.js:7]: Yes! I'll update it. For now I only have updated the Components' styles.

File: src/Components/CustomMap/CustomMap.js (hunk 1 above)
Comment (cfrancisco) [reply src/Components/CustomMap/CustomMap.js]: It's already mapId, this file is outdated.

File: src/Pages/Dashboard/Dashboard.js (hunk 2 above)
Comment (cfrancisco) [reply src/Pages/Dashboard/Dashboard.js:116]: In my opinion, that could be made to methods inside the components aiming a more reusable code. Pages, in general, sound to me as a specific code to each application.

**Pull request merged**